<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>494</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>rcontact</string>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="ListeContacts" name="listeContacts" native="true"/>
    </item>
    <item>
     <widget class="EditerContact" name="editerContact" native="true"/>
    </item>
    <item>
     <widget class="AfficherContact" name="afficherContact" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouvelle_liste"/>
    <addaction name="actionNouvelle_personne"/>
    <addaction name="actionNouveau_organisme"/>
    <addaction name="actionCharger"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionTrier"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionRechercher"/>
    <addaction name="actionFinir_la_recherche"/>
    <addaction name="actionSupprimer"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouvelle_liste"/>
   <addaction name="actionNouvelle_personne"/>
   <addaction name="actionNouveau_organisme"/>
   <addaction name="actionCharger"/>
   <addaction name="actionEnregistrer"/>
   <addaction name="actionTrier"/>
   <addaction name="actionRechercher"/>
   <addaction name="actionFinir_la_recherche"/>
   <addaction name="actionSupprimer"/>
   <addaction name="actionQuitter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouvelle_personne">
   <property name="text">
    <string>Nouveau personne</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="text">
    <string>Charger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRechercher">
   <property name="text">
    <string>Rechercher</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSupprimer">
   <property name="text">
    <string>Supprimer</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionTrier">
   <property name="text">
    <string>Trier</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFinir_la_recherche">
   <property name="text">
    <string>Finir la recherche</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionNouveau_organisme">
   <property name="text">
    <string>Nouveau organisme</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+T</string>
   </property>
  </action>
  <action name="actionNouvelle_liste">
   <property name="text">
    <string>Nouvelle liste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EditerContact</class>
   <extends>QWidget</extends>
   <header>vue/editercontact.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AfficherContact</class>
   <extends>QWidget</extends>
   <header>vue/affichercontact.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListeContacts</class>
   <extends>QWidget</extends>
   <header>vue/listecontacts.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
